packages/sdk/src/transactions/types.ts

```diff
@@ -0,0 +1,166 @@
+import type { Inscription, Ordinal } from "../inscription/types";
+
+// Types are based on bitcoin RPC
+// Refer to https://bitcoincore.org/en/doc/25.0.0/rpc/rawtransactions/decoderawtransaction/
+
+/**
+ * Transaction output (Vout)
+ */
+export type Vout = {
+  /**
+   * Value in BTC
+   */
+  value: number;
+
+  /**
+   * Index
+   */
+  n: number;
+
+  /**
+   * Ordinals
+   */
+  ordinals: Ordinal[];
```

Comment:
@iamcrazycoder could you help link where this is from?

Reply:
Part of the [Transactions.GetTransaction](https://sado.space/docs/api-transactions#:~:text=blockchain%20related%20data.-,Get%20Transaction,-Get%20a%20transaction) RPC response. Documentation doesn't mention it though.